ugvc/mrd/mrd_utils.py

```diff
@@ -772,3 +777,75 @@ def concat_dataframes(dataframes: list, outfile: str, n_jobs: int = 1):
         df = df.reset_index()
     df.to_parquet(outfile)
     return df
+
+
+def generate_synthetic_signatures(
+    signature_vcf: str, db_vcf: str, n_synthetic_signatures: int, output_dir: str
+) -> list[str]:
+    """
+    Generate synthetic signatures from a signature vcf file and a db vcf file
+
+    Parameters
+    ----------
+    signature_vcf: str
+        signature vcf file
+    db_vcf: str
+        db vcf file
+    n_synthetic_signatures: int
+        number of synthetic signatures to generate
+    output_dir: str
+        output directory
+
+    Returns
+    -------
+    synthetic_signatures: list[str]
+        list of synthetic control vcf files
+
+    """
```

Comment:
I think you should add a "random_seed:int = 0" argument and do "np.random.seed(random_seed)" in the first line, to make sure that the output is determinstic. Specifically for your test, which I suspect could fail if it's not.

Reply:
Done